src/qibojit/backends/cpu.py

```diff
@@ -28,11 +28,20 @@ def __init__(self):
         import sys
 
         import psutil
+        from numba import __version__ as numba_version
+        from qibo import __version__ as qibo_version
 
+        from qibojit import __version__ as qibojit_version
         from qibojit.custom_operators import gates, ops
 
         self.name = "qibojit"
         self.platform = "numba"
+        self.versions = {
+            "qibo": qibo_version,
+            "qibojit": qibojit_version,
+            "numpy": self.np.__version__,
+            "numba": numba_version,
+        }
```

Comment:
If you want to save some lines you can just do
```suggestion
        self.versions.update({
            "qibojit": qibojit_version,
            "numba": numba_version,
        })
```
as the qibo and numpy versions will be created by the NumpyBackend during the `super().__init__()` call.

Reply:
Thanks, I agree.